qemu/tests/cfg/balloon_uniqueness.cfg

```diff
@@ -0,0 +1,17 @@
+- balloon_uniqueness:
+    type = balloon_uniqueness
+    balloon_dev_add_bus = 'yes'
+    variants:
+        - boot_with_two:
+            start_vm = no
+            balloon = balloon0 balloon1
+            balloon_dev_devid_balloon0 = balloon0
+            balloon_dev_devid_balloon1 = balloon1
+            boot_warning_msg = 'Only one balloon device is supported'
```

Comment:
It's more like an error than a warning since qemu just refuse to start.

Reply:
@huangyum You mean name it `boot_error_msg`?

Reply:
Yeah, that one looks better.